src/analysis/retail/warlock/demonology/modules/talents/DiabolicRitual.tsx

```diff
@@ -0,0 +1,64 @@
+import { formatThousands } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/warlock';
+import Analyzer, { Options, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+class DiabolicRitual extends Analyzer {
+  felseeker_damage = 0;
+  wicked_cleave_damage = 0;
+  chaos_salvo_damage = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.DIABOLIC_RITUAL_TALENT);
+    this.addEventListener(
+      Events.damage.by(SELECTED_PLAYER_PET).spell(SPELLS.FELSEEKER_DAMAGE),
+      this.handleFelseekerDamage,
+    );
+    this.addEventListener(
+      Events.damage.by(SELECTED_PLAYER_PET).spell(SPELLS.WICKED_CLEAVE_DAMAGE),
+      this.handleWickedCleaveDamage,
+    );
+    this.addEventListener(
+      Events.damage.by(SELECTED_PLAYER_PET).spell(SPELLS.CHAOS_SALVO_DAMAGE),
+      this.handleChaosSalvoDamage,
+    );
+  }
+
+  handleFelseekerDamage(event: DamageEvent) {
```

Comment:
Since you're never differentiating between the 3 damage types you could have it summed into a single damage variable and just do something like 
```
 this.addEventListener(
      Events.damage.by(SELECTED_PLAYER_PET).spell([SPELLS.CHAOS_SALVO_DAMAGE, SPELLS.FELSEEKER_DAMAGE, SPELLS.WICKED_CLEAVE_DAMAGE]),
      this.handleDamage,
    );
```

Reply:
thanks, i don't know i can pass an array in !